lib/galaxy/objectstore/templates/examples/production_onedata.yml

```diff
@@ -0,0 +1,46 @@
+- id: onedata
+  name: Onedata Storage
+  version: 0
+  description: |
+    This template allows using Onedata storage.
+
+    Onedata is a global data management system providing a unified data access
+    interface across different storage providers. More information on Onedata
+    can be found in [Onedata's website](https://onedata.org/#/home).
+  variables:
+    onezone_domain:
+      label: Onezone Domain
+      type: string
+      help: Domain of the Onezone service (e.g. datahub.egi.eu) to connect to.
+    disable_tls_certificate_validation:
+      label: Disable tls certificate validation?
+      type: boolean
+      help: |
+        Allows connection to Onedata servers that do not present trusted SSL certificates.
+        SHOULD NOT be used unless you really know what you are doing.
+      default: False
+    space_name:
+      label: Space Name
+      type: string
+      help: The name of the Onedata space where the Galaxy data will be stored.
```

Comment:
teraz pomyślałem że może ktoś mieć kolizje nazw, więc do rozważenia jak to rozegrać, pomysły:
- pozwolić podawać tutaj ID
- pozwolić podawać tutaj FQN (pewnie to zadziała od kopa? trzeba by opisać w helpie że tak można)